packages/dds/tree/src/feature-libraries/defaultFieldKinds.ts

```diff
@@ -532,29 +605,63 @@ const optionalChangeRebaser: FieldChangeRebaser<OptionalChangeset> = isolatedFie
 			if (over.fieldChange !== undefined) {
 				const wasEmpty = over.fieldChange.newContent === undefined;
 
-				// We don't have to rebase the child changes, since the other child changes don't apply to the same node
+				// TODO: Handle rebasing child changes.
 				return {
 					...change,
 					fieldChange: { ...change.fieldChange, wasEmpty },
 				};
 			}
 
-			return change;
+			const rebasedChange = { ...change };
+			const overChildChange =
+				change.deletedBy === over.deletedBy ? over.childChange : undefined;
```

Comment:
In the case where `change.deletedBy` and `over.deletedBy` are defined and equal, don't we want to also have `overChildChange` be `undefined`?
The `over.childChange` are muted in that case so rebasing over them should be no-op.

Reply:
Would be good to add a test case for that (assuming my comment is correct).

Reply:
Update: Alex and I discussed about different models for what it could mean to edit a node that has become unreachable because of a concurrent delete. We have yet to come to a conclusion, but it's looking likely we'll adopt the semantics implemented here.